.github/workflows/ci-knip.yml

```
@@ -0,0 +1,37 @@
+      runs-on: ubuntu-latest
+      env:
+        NODE_OPTIONS: "--max_old_space_size=4096"
+      name: Run Knip Linter
+      if: github.event.pull_request.head.repo.full_name == github.repository
+      steps:
+        - name: Checkout Repo
+          uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+          with:
+            ref: ${{ github.head_ref }}
+            token: ${{ secrets.STUDIOCMS_SERVICE_TOKEN }}
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)